it/test/uk/gov/hmrc/tradergoodsprofilesdatastore/connectors/RouterConnectorSpec.scala

```diff
@@ -111,4 +113,38 @@ class RouterConnectorSpec
       connector.getRecords(eori, Some(lastUpdatedDate), Some(page), Some(recordsize)).failed.futureValue
     }
   }
+
+  ".deleteRecord" - {
+
+    "must delete record from from B&T database" in {
+
+      wireMockServer.stubFor(
+        delete(
+          urlEqualTo(
+            s"/trader-goods-profiles-router/traders/$eori/records/$recordId?actorId=$actorId"
+          )
+        )
+          .withHeader("X-Client-ID", equalTo("tgp-frontend"))
+          .willReturn(noContent())
+      )
+
+      connector.deleteRecord(eori, recordId, actorId).futureValue
+    }
+
+    "must return a failed future when the server returns an error" in {
+
+      wireMockServer.stubFor(
+        delete(
+          urlEqualTo(
+            s"/trader-goods-profiles-router/traders/invalid-eori/records/invalid-recordId?actorId=$actorId"
+          )
+        )
+          .withHeader("X-Client-ID", equalTo("tgp-frontend"))
+          .willReturn(badRequest())
```

Comment:
should this be serverError instead of badRequest? Or maybe you need a test or both - but that could be overkill

Reply:
You are right :)